### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by IgorSantos17**
- Added new elephpants
- Configs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+### Stack: 
+
+#### Frontend
+- HTML5
+- CSS3
+- Bootstrap
+- JavaScript
+- JQuery
+
```
Comment (jgrossi) [inline README.md]: add Stimulus.js please

File: .gitignore [hunk 2]
```diff
@@ -1,12 +1,33 @@
...
 /storage/*.key
 /vendor
+/.vagrant
+/storage/logs
+
+# Laravel Files
 .env
-.env.backup
-.phpunit.result.cache
+package-lock.json
```
Comment (jgrossi) [inline .gitignore]: remove please

File: .gitignore [hunk 3]
```diff
@@ -1,12 +1,33 @@
...
+composer.lock
```
Comment (jgrossi) [inline .gitignore]: remove please

File: .gitignore [hunk 4]
```diff
@@ -1,12 +1,30 @@
...
-.env.backup
-.phpunit.result.cache
 Homestead.json
 Homestead.yaml
 npm-debug.log
 yarn-error.log
+.phpunit.result.cache
+
+# Git Files
+.git
```
Comment (jgrossi) [inline .gitignore]: remove please

File: README.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+#### Backend
+- PHP 7.2
```
Comment (jgrossi) [inline README.md]: 7.2+

File: README.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+- Laravel 6
+- Composer
+- PHPUnit
+
+#### Database
+- MySQL
```
Comment (jgrossi) [inline README.md]: 5.8+

File: app/Providers/AppServiceProvider.php [hunk 7]
```diff
@@ -23,6 +24,7 @@ public function register()
      */
     public function boot()
     {
-        //
+        Schema::defaultStringLength(191);
```
Comment (jgrossi) [inline app/Providers/AppServiceProvider.php]: this is only required for MySQL 5.7, so please remove

File: app/Providers/AppServiceProvider.php [hunk 8]
```diff
@@ -23,6 +24,7 @@ public function register()
...
+        setlocale(LC_TIME, config('app.locale'));
```
Comment (jgrossi) [inline app/Providers/AppServiceProvider.php]: this is already done by Laravel's core

File: README.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
+## ElePHPant.me
+
+> Here is the right place for your elePHPants!
+
+
+### About
+- You can add your herd
+- See ranking
+    - global / per country
+- Find people to tradecação
```
Comment (jgrossi) [inline README.md]: typo?

File: README.md [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+
+#### Backend
+- $ composer install  
+- $ php artisan key:generate
+- $ php artisan elephpants:read
+- $ php artisan storage:link
+
+#### Database
+- $ php artisan migrate
+- $ php artisan db:seed
```
Comment (jgrossi) [inline README.md]: please add "(only for generating fake data locally)"

File: README.md [hunk 11]
```diff
@@ -0,0 +1,55 @@
...
+
+---
+
+### Responsible
```
Comment (jgrossi) [inline README.md]: change to "Maintainers" please

**Comment by IgorSantos17** [comment]
All Resolved

**Pull request merged**